# Constants.
uint8 BUTTON_RIGHT = 1
uint8 BUTTON_LEFT = 2

uint8 STATE_RELEASED = 0
uint8 STATE_PRESSED = 1

# Fields.
uint8 button
uint8 state

